.github/workflows/gradle.yml

```diff
@@ -33,7 +33,7 @@ jobs:
   release:
     name: Publish artifacts
     runs-on: ubuntu-latest
-    if: github.event_name != 'pull_request'
+    if: github.event_name != 'pull_request' && github.repository == 'Discord4J/Discord4J'
```

Comment:
Is this needed? I'm thinking about pushing some changes to avoid some issues we're seeing and want to avoid having a conflict.

Reply:
removed